### DISCUSSION TITLE
Ideas para logo

### DISCUSSION THREAD

**Discussion by BraianTroncoso** [DebuggingDemons/Integrador_Tercer_Semestre#14] [Ideas]
# Adjunten ideas, diseños de logos, referencias de webs.

**Comment by SebasGalvan** [comment]
Dall e sugirió esto 
![image](https://github.com/DebuggingDemons/Integrador_Tercer_Semestre/assets/47829817/a8f215bc-c311-4aa0-a943-4e4818639dd4)
